### ISSUE TITLE
make distclean fails

### ISSUE THREAD

**Issue by olifre** [apptainer/singularity#1206]
## Version of Singularity:

development branch

### Expected behavior

`make distclean` runs fine after `configure`.

### Actual behavior

It fails.

### Steps to reproduce behavior
```
./autogen.sh
./configure --prefix=/usr/local
make distclean
```

This is tested now as part of https://github.com/singularityware/singularity/pull/1204 and usually called by `debuild`. 
The travis report linked to the mentioned PR shows the encountered errors, here a short excerpt:
```
make[5]: Leaving directory '/tmp/debian9/singularity/src/lib/runtime/autofs'
make[5]: Entering directory '/tmp/debian9/singularity/src/lib/runtime'
Makefile:540: ../../util/.deps/libinternal_la-capability.Plo: No such file or directory
Makefile:541: ../../util/.deps/libinternal_la-config_parser.Plo: No such file or directory
Makefile:542: ../../util/.deps/libinternal_la-file.Plo: No such file or directory
Makefile:543: ../../util/.deps/libinternal_la-fork.Plo: No such file or directory
Makefile:544: ../../util/.deps/libinternal_la-message.Plo: No such file or directory
Makefile:545: ../../util/.deps/libinternal_la-mount.Plo: No such file or directory
Makefile:546: ../../util/.deps/libinternal_la-privilege.Plo: No such file or directory
Makefile:547: ../../util/.deps/libinternal_la-registry.Plo: No such file or directory
Makefile:548: ../../util/.deps/libinternal_la-setns.Plo: No such file or directory
Makefile:549: ../../util/.deps/libinternal_la-suid.Plo: No such file or directory
Makefile:550: ../../util/.deps/libinternal_la-util.Plo: No such file or directory
make[5]: *** No rule to make target '../../util/.deps/libinternal_la-util.Plo'.  Stop.
make[5]: Leaving directory '/tmp/debian9/singularity/src/lib/runtime'
Makefile:696: recipe for target 'distclean-recursive' failed
```
Since I'm no automake expert (I usually use more modern and nowadays more widespread build systems such as cmake or meson), I sadly can't offer a fix.

**Comment by jmstover** [comment]
This comes from the Makefile being generated in `src/lib/runtime` including lines like:

```
include ../../util/$(DEPDIR)/libinternal_la-capability.Plo
```

I'm messing around with the `Makefile.am` to see what I can come up with.

**Comment by dtrudg** [comment]
Closing - this was fixed in #1246
